internal/controller/nnf_workflow_controller_helpers.go

```diff
@@ -1535,6 +1540,21 @@ func (r *NnfWorkflowReconciler) waitForContainersToStart(ctx context.Context, wo
 			}
 		}
 
+		// If we're up and running and this is copy offload, then determine which rabbit node the
+		// launcher pod landed on and set an env var for flux. This tells the compute how to contact
+		// the copy offload server.
+		if running && slices.Contains(workflow.Status.Requires, requiresCopyOffload) {
+			jobList, err := r.getMPIJobChildrenJobs(ctx, workflow, mpiJob)
+			if err != nil || len(jobList.Items) < 1 {
+				return nil, dwsv1alpha3.NewResourceError("could not retrieve MPIJob Child Jobs to find launcher node").WithError(err).WithFatal()
+			}
+
+			launcherJob := jobList.Items[0]
+			launcherNnfNode := launcherJob.Spec.Template.Spec.NodeSelector["kubernetes.io/hostname"]
+			workflow.Status.Env["NNF_COPY_OFFLOAD_SERVER"] = launcherNnfNode
```

Comment:
So far, the plan is that the libcopyoffload library knows how to figure out which rabbit to talk to. Up to now, that means looking at a special file on the compute to read the rabbit's name.
With this PR, I think we want libcopyoffload to first look for this environment variable and use it, if found. If not found, then revert to looking for that special file.

Reply:
In that case, let's make this env var less specific to copy-offload. Let's name it "NNF_USER_SERVER".

Reply:
In this particular case though, it's only set when `requires=copy-offload`.

Reply:
We probably need the env var any time the container profile uses an mpiJob.

Reply:
Agreed. Perhaps we do both: Have a generic one for `NNF_CONTAINER_LAUNCHER` and also keep `NNF_COPY_OFFLOAD_SERVER` for this case to make it easy on the user.

But based on your comment above, maybe it doesn't need to be easy if the library defaults to having the environment variable.

Reply:
Just one, something that makes sense to someone writing an app for their user container.
For copy-offload, the library will know how to get it right and the user won't need to fuss with it.